scripts/iast/leak_functions.py

```diff
@@ -34,14 +48,19 @@ def test_iast_leaks(iterations: int, fail_percent: float, print_every: int):
         print("Test %d iterations" % iterations)
         current_rss = 0
         half_rss = 0
+        enable_iast_propagation()
+        from scripts.iast.mod_leak_functions import test_doit
```

Comment:
The import and the check probably can be outside of the loop since now the script is significatly slower, this could help making it faster.